src/idl_gen_general.cpp

```diff
@@ -203,8 +203,13 @@ class GeneralGenerator : public BaseGenerator {
       const std::string &classcode, bool needs_includes) {
     if (!classcode.length()) return true;
 
-    std::string code;
-    code = code + "// " + FlatBuffersGeneratedWarning();
+    std::string code("//------------------------------------------------------------------------------\n"
```

Comment:
I Just found the xsd.exe comment is generated. It is in [system.dll](https://github.com/Microsoft/referencesource/blob/4fe4349175f4c5091d972a7e56ea12012f1e7170/System/compmod/microsoft/csharp/csharpcodeprovider.cs#L2883)

So it will be used by all tools that generate source with the class CSharpCodeGenerator